app/controllers/idv/finance_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Idv
+  class FinanceController < ApplicationController
+    include IdvSession
+
+    before_action :confirm_two_factor_authenticated
+    before_action :confirm_idv_session_started
+
+    def new
+      prep_vars
+    end
+
+    def create
+      prep_vars
+
+      if @idv_finance_form.submit(finance_params)
+        self.idv_params = @idv_finance_form.idv_params
+        redirect_to idv_phone_url
+      else
+        render :new
+      end
+    end
+
+    private
+
+    def prep_vars
+      @idv_params = idv_params
```

Comment:
I find the presence of `@idv_params` and `#idv_param` a little confusing here.   I'd also worry they could get out of sync.

Seems like `@idv_params` is just needed for the views, right?   Could you instead do `helper_method :idv_params` and remove the `@idv_params` completely?